src/language-js/utils.js

```diff
@@ -443,16 +443,28 @@ const flowTypeAnnotations = new Set([
   "BooleanLiteralTypeAnnotation",
   "StringTypeAnnotation",
 ]);
+const simpleTypeScriptTypeAnnotations = new Set([
+  "TSAnyKeyword",
+  "TSNullKeyword",
+  "TSTypeReference",
+  "TSThisType",
+  "TSNumberKeyword",
+  "TSVoidKeyword",
+  "TSBooleanKeyword",
+  "TSLiteralType",
```

Comment:
I think we should add `"TSSymbolKeyword"`.
**Prettier pr-9521**
[Playground link](https://deploy-preview-9521--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuc0DOMAEAzCEmYBieJpZ5FlV1NpAPGgJ4C2ARhADYB8A3ADpQQAGhAQADjACW6ZKACGAJwUQA7gAVFCNMhByOKuY20jWCuWADWcGAGUx5yVADmyGAoCucEXDZwAJn7+ADJyzu5yTnDECsxyMFLOOnLuMBDCIAAWMMwcAOoZkvBo9mBwNlqFkgBuhYw6YGjGII5ocAowamZOscjYeq0iAFZoAB4AQmaW1jZyzHBBjnC9-V4gwyM2jk4ccACK7hDwyxwDIPYKrQo6MIxicGhgCpIS6WJPsLmSfjAZyAAcAAYRG8IK1cmYxDo3vc2lUliIAI4HeCdcTaFByNAAWigcH8-nSCjgSMkRM6ER6SD6J1WrWYklcHlpWx2+0OSypKxEMDkrE+31+SAATNyzJIOFsAMIQZiUkD3ACs6XcrQAKrz0dTTlVPABJKCBWA2R7PGAAQQNNhuO2OrQAvnagA)
```sh
--parser typescript
```

**Input:**
```tsx
const foo: Fooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo<symbol>;

```

**Output:**
```tsx
const foo: Fooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo<
  symbol
>;

```

Reply:
Flow format it like this

**Prettier pr-9521**
[Playground link](https://deploy-preview-9521--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuc0DOMAEAzCEmYBieJpZ5FlV1NpAPADpSaZoCeAtgEYQA2TAPkwBeTAEMA3CAA0ICAAcYAS3TJQYgE4aIAdwAKmhGmQgxvHWLbHZXDWLABrODADK8+0qgBzZDA0BXOFk4bjgAEzDwgBkxb38xLzhiDQ4xGGVvEzF-GAgZEAALGA5eAHUCpXg0dzA4FyNKpQA3SrYTMDRrEE80OA0YPTsvVORsM17ZACs0AA8AITtHZxcxDjgozzhR8aCQaZmXTy9eOABFfwh4bd4JkHcNXo0TbF5dfPkNTxhSpTCYAuQAA4AAyyD4QXqlOzyEwfOCPJpbWQARwu8EGCmMKDEaAAtFA4OFwvkNHBUUpSYMEiMkGMbrtehwlL4AgyjidzpctrSdrIYGIuD8-gCkAAmPl2JS8I4AYQgHBpIHhAFZ8v5egAVAVYum3JqBACSUEisBcYE+igAgsaXDA2Cdrr0AL5OoA)
```sh
--parser flow
```

**Input:**
```jsx
const foo: Fooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo<
  symbol
> = a;
```

**Output:**
```jsx
const foo: Fooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo<
  symbol
> = a;

```

I know this is just two types into the list, but I still prefer in another PR.
I added a test for it https://github.com/prettier/prettier/pull/9521/commits/a21ebc28d968c858c41861f54d1fef56ef75f0ff